scraping/spiders/canada_immigration_services.py

```diff
@@ -0,0 +1,99 @@
+import re
+import logging
+
+import scrapy
+import hjson
+
+
+log = logging.getLogger(__name__)
+
+
+DEFAULT_CATEGORY = 'Any Help'
+
+CATEGORIES = {
+    'help_gar': 'Accommodation',
+    'lgbtq2': 'Social help',
+    'youth': 'Children',
+    'general': 'Any help',
+    'other': 'Any help',
+}
+
+# Ref: https://ircc.canada.ca/js/tools/services/closest-services.js
+SERVICES = {
+    'Lang_assess': 'Language assessment',
+    'Lang_training': 'Language training (general)',
+    'Job_lang_training': 'Job-specific language training',
+    'General': 'Help with daily life',
+    'Job_search': 'Help finding a job',
+    'Citzenship_test_prep': 'Help preparing for the citizenship test',
+    'Volunteer': 'Find or become a mentor to a newcomer',
+    'Help_gar': 'Services for refugees',
```

Comment:
I think you should not scrape everything but first pre-filter everything by `Services for refugees` and ignore all other filters

Reply:
Ah, okay, let me filter out all the rest. I'll keep the category mappings though if they make sense.